src/components/AccountLoginForm.jsx

```diff
@@ -26,6 +26,7 @@ const AccountLoginForm = ({ t, isOAuth, customView, fields, error, dirty, submit
               name={inputName}
               placeholder={t('account.connection.login.password.placeholder')}
               invalid={!!error}
+              noPasswordSave
```

Comment:
I think you would be right if autocomplete="new-password" was in the standard, which is not the case. This is a new non standard value to cancel a non standard behavior of recent browsers.